addons/website_slides/models/slide_channel.py

```diff
@@ -765,6 +765,10 @@ def _remove_membership(self, partner_ids):
 
     def _send_share_email(self, emails):
         """ Share channel through emails."""
+        courses_without_templates = self.filtered(lambda channel: not channel.share_channel_template_id)
+        if courses_without_templates:
+            raise UserError(_('Impossible to send emails. Select a "Channel Share Template" for the following courses first:')
```

Comment:
Maybe we could get rid of the '\n' and avoid string concatenation (easier for translations), with something like
```
raise UserError(
    _('Impossible to send emails. Select a "Channel Share Template" for courses %(course_name)s first',
      course_names=', '.join(courses_without_templates.mapped('name'))
    )
)
```